plugins/MantisCoreFormatting/files/markdown.css

```diff
@@ -1,3 +1,24 @@
+/*
+ * Rules taken from bootstrap.
+ *
+ * The right way is creating boostrap css from scss sources instead
+ * of override the rules here.
+ *
+ * @todo fix this directly in bootstrap and delete this rules.
```

Comment:
I'm not sure what you mean by fixing directly in Bootstrap... Opening an issue to have it fix upstream ? Considering that we are using a fairly old version of that library, I don' think it's worth bothering as it would certainly not be be accepted.

I'd rather keep the bundled bootstrap files untouched, an override of specific CSS rules like here is acceptable to me.

Grammar fixes: 
- instead of **overriding**
- delete **these** rules

Reply:
> I'd rather keep the bundled bootstrap files untouched,

Of course.

That was misleading, I meant that it is the normal way to customize the bootstrap css via the variables or to remove parts from the scss sources that you don't need. 
The idea was to do it this way, i.e. set the color via the variables and rebuild the css with it. But now I realized that bootstrap is just way too old for that. 

The comment is removed.